src/Run/Constants.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.DotNet.Execute
+{
+    public struct OS
+    {
+        public const string OSX = "OSX";
+        public const string FreeBSD = "FreeBSD";
+        public const string NetBSD = "NetBSD";
+        public const string Linux = "Linux";
+        public const string Unix = "Unix";
+        public const string Windows = "Windows_NT";
```

Comment:
Yeah, currently we don't have a OS Name for specific windows versions in the build system, all windows builds have /p:OSGroup=Windows_NT.